content/rosa/customizing-console-route/index.md

```diff
@@ -0,0 +1,123 @@
+---
+date: '2023-11-30'
+title: Customizing the console URL in ROSA
+tags: ["ROSA", "AWS"]
+authors:
+  - Charlotte Fung
+  - Thatcher Hubbard 
+---
+
+Starting with ROSA 4.14.X, it is possible to modify the hostname and TLS certificate of component Routes post-install. These are the `OAuth`, `Console`, and `Downloads` routes. For example, the default ROSA console uses the built-in domain `https://console-openshift-console.apps.<cluster_name>.<random>.p1.openshiftapps.com`. You can now specify a custom domain, for example `test.example.com`, and the ROSA console will be available at a URL such as `https://console-openshift-console.test.example.com`.   
+
+## Prerequisites
+
+* A Red Hat OpenShift for AWS (ROSA) cluster 4.14.0 or higher
+* Cluster admin privileges (required to be able to modify cluster ingress)
+* Route53 Hosted Zone
+* A custom domain
+* A wildcard certificate 
+* `rosa` cli
+* `oc` cli
+
+## Deploy 
+
+### Set up Environment
+
+1. Set your email and domain
+    
+     ```bash
+     export EMAIL=<YOUR-EMAIL>
+     export DOMAIN=<YOUR-DOMAIN>
+     ```
+
+1. Set remaining environment variables 
+   
+     ```bash
+     export SCRATCH_DIR=/tmp/scratch
+     export ZONE_ID=$(aws route53 list-hosted-zones-by-name --output json \
```

Comment:
where is this variable being used ?

Reply:
Oops, I will remove the ZONE_ID variable. I don't use that any where in the doc